test/components/views/rooms/RoomTile-test.tsx

```diff
@@ -285,7 +305,7 @@ describe("RoomTile", () => {
         });
 
         it("should render a room without a message as expected", async () => {
-            renderRoomTile();
+            const renderResult: RenderResult = renderRoomTile();
```

Comment:
Can we ditch these explicit types? `renderRoomTile` has a clear return type, this just makes refactors more messy

Reply:
fixed, thanks for issue